package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^12.1.4",
     "@testing-library/user-event": "^13.5.0",
+    "react-router-dom": "^4.2.2",
     "big-js": "^3.1.3",
     "big.js": "^6.1.1",
     "prop-types": "^15.8.1",
```

Comment:
Kindly remove this line of code from your `packages.json' file to enable your app to run correctly.

![image](https://user-images.githubusercontent.com/10332499/159833278-2ee36410-9410-434b-b547-f267fae09794.png)